datamade/la-metro-dashboard
# Pull images from GHCR

## Overview

This PR updates the dashboard to pull images from GitHub Container Registry instead of Dockerhub. It also migrates from using `staging` and `production` image tags to `master` and `deploy` image tags (because these were easier to automate through GitHub Actions).

### Checklist


## Testing Instructions

 * Merge to deploy to staging. Confirm both a scrape and ETL DAG run successfully.
 * Deploy Metro app to create a `deploy` tag of the app image.
 * Deploy dashboard to production and confirm both a scrape and ETL DAG run successfully.
